library/menu/src/PatientMenuRole.php

```diff
@@ -0,0 +1,67 @@
+<?php
+/**
+ * MainMenuRole class.
+ *
+ * @package OpenEMR
+ * @link    http://www.open-emr.org
+ * @author  Brady Miller <brady.g.miller@gmail.com>
+ * @copyright Copyright (c) 2017 Brady Miller <brady.g.miller@gmail.com>
+ * @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+namespace OpenEMR\Menu;
+
+use OpenEMR\Services\UserService;
+
+class PatientMenuRole
+{
+
+    /**
+     * Collect the patientMenuRole for logged in user.
+     *
+     * @return string Identifier for the patientMenuRole
+     */
+    public static function getPatientMenuRole()
+    {
+        $userService = new UserService();
+        $user = $userService->getCurrentlyLoggedInUser();
+        $patientMenuRole = $user->getPatientMenuRole();
+        if (empty($patientMenuRole)) {
+            $patientMenuRole = "standard";
+        }
+
+        return $patientMenuRole;
+    }
+
+    /**
+     * Build the html select element to list the MainMenuRole options.
+     *
+     * @var string $selected Current MainMenuRole for current users.
+     * @return string Html select element to list the MainMenuRole options.
+     */
+    public static function displayPatientMenuRoleSelector($selected = "")
+    {
+        $output = "<select name='patient_menu_role' id='patient_menu_role' class='form-control'>";
+        $output .= "<option value='standard' " . (($selected == "standard") ? "selected" : "") . ">" . xlt("Standard") . "</option>";
+
+        $customMenuDir = $GLOBALS['OE_SITE_DIR'] . "/documents/custom_menus/patient/menus/";
```

Comment:
use patient_menus in path and not patient/menus

Reply:
ok